cmd/ci-operator/main.go

```diff
@@ -241,6 +245,7 @@ func bindOptions(flag *flag.FlagSet) *options {
 	flag.StringVar(&opt.gitRef, "git-ref", "", "Populate the job spec from this local Git reference. If JOB_SPEC is set, the refs field will be overwritten.")
 	flag.BoolVar(&opt.givePrAuthorAccessToNamespace, "give-pr-author-access-to-namespace", false, "Give view access to the temporarily created namespace to the PR author.")
 	flag.StringVar(&opt.impersonateUser, "as", "", "Username to impersonate")
+	flag.BoolVar(&opt.exposeTemplatePod, "expose-template-pod", false, "Expose the template's pod by creating a service and two routes for ports 8080 and 8443.")
```

Comment:
routes are actually for ports 80 and 443, but end up being redirected as 8080 and 8443 to the pod itself